docs/DeveloperGuide.md

```diff
@@ -178,6 +178,10 @@ in the list of instruments:
 
 <img src="images/DoneCryptoSequenceDiagram.png" width="1040"/>
 
+More details about the reference frame for executing the done command is shown below:
+
+<img src="images/DoneCryptoExecuteDiagram.png" width="600"/>
+
 ### FileManager Component
 The `filemanager` package contains the `Storage`, `InstrumentEncoder` and `InstrumentDecoder` classes. It is defined in
```

Comment:
The XYZDecoder shld not have addStockToList but addXYZToList. And make the diagram bigger. Make the model diagram smaller.